lib/saluki-components/src/sources/checks/mod.rs

```diff
@@ -236,24 +289,57 @@ async fn drain_and_dispatch_check_metrics(
                 debug!("Checks events listerners received shutdown signal.");
                 break;
             }
-            result = check_metrics_rx.recv() => match result {
+            result = check_event_rx.recv() => match result {
                 None => break,
-                Some(check_metric) => {
-                    let mut buffered_dispatcher = context
-                        .dispatcher()
-                        .buffered()
-                        .expect("checks source should always have default output");
+                Some(check_event) => {
+                    trace!("Received check event: {:?}", check_event);
+
+                    match check_event.event_type() {
+                       EventType::Metric => {
+                            let mut buffered_dispatcher = context
+                                .dispatcher()
+                                .buffered_named("metrics")
+                                .expect("checks source should always have default output");
```

Comment:
This message doesn't make sense anymore since it's a named output... same with the other equivalent calls for service checks and events.